nodes/modifier_change/rigid_origami.py

```diff
@@ -0,0 +1,130 @@
+# This file is part of project Sverchok. It's copyrighted by the contributors
+# recorded in the version control history of the file, available from
+# its original location https://github.com/nortikin/sverchok/commit/master
+#
+# SPDX-License-Identifier: GPL3
+# License-Filename: LICENSE
+
+import bpy
+from bpy.props import IntProperty, FloatProperty
+from sverchok.data_structure import updateNode, match_long_repeat
+from sverchok.node_tree import SverchCustomTreeNode
+from sverchok.utils.rigid_origami_utils import ObjectParams, \
+		CreaseLines, InsideVertex, FoldAngleCalculator, FaceRotation
+
+class SvRigidOrigamiNode(bpy.types.Node, SverchCustomTreeNode):
+    """
+    Triggers: Rigid Origami
+    Tooltip: Fold a paper like a rigid origami
+    """
+
+    bl_idname = 'SvRigidOrigamiNode'
+    bl_label = 'Rigid Origami'
+    bl_icon = 'OUTLINER_OB_EMPTY'
+    sv_icon = 'SV_ORIGAMI'
+
+    folding_ratio : FloatProperty(
+        name = "Folding ratio",
```

Comment:
according to PEP8  

> Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter

so: 
```
name="Folding ratio"
description="Folding ratio from 0.0 to 1.0" 
```

and so on

Reply:
Thank you for the suggestion, and sorry that I had missed it. I modified the places which has the same issue at https://github.com/nortikin/sverchok/pull/3416/commits/2d0560d679e47680fad664d99c79d1b42ceb8880 .